functional/adoptopenjdk/build.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
```

Comment:
Any reason for Ant over Gradle or Maven?

Reply:
I'm not aware that I could use anything other than Ant. Would be happy to use Gradle because that would hopefully allow IntelliJ to run the tests locally.

Reply:
Please do not add new prereqs.  If/when we would be in a position to remove ant as a prereq that allows us to compile even some of the older test material we incorporated into AQA, then we could consider whole-hearted replacement, but that would have to be planned and scheduled and would have to bump other items down the priority list.  

It is not a requirement to use a different tool for building test material, and not something we have resources for doing at this time (safely and without destabilizing what we have at present).  I do not want a blend of solutions that increases the different ways we do things and increases the number of items that have to be installed on a machine before people can 'get testing'... I'd want to move towards decreasing the differences in how things are built and executed (if we add a prereq, we should be removing a prereq).